File: tests/conversion/fixtures/ts/StringTypes.ts
export class StringTypes {
    private name: string;
    
    public constructor(name: string) {
        this.name = name;
    }
    
    public getName(): string {
        return this.name;
    }
    
    public setName(name: string): void {
        this.name = name;
    }
    
    public formatGreeting(prefix: string): string {
        return prefix + " " + this.name;
    }
}

File: tests/conversion/fixtures/java/StringTypes.java
public class StringTypes {
    private String name;
    
    public StringTypes(String name) {
        this.name = name;
    }
    
    public String getName() {
        return name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
    
    public String formatGreeting(String prefix) {
        return prefix + " " + name;
    }
}
